view/frontend/templates/checkout/onepage/success/tesco_additional_info.phtml

```diff
@@ -0,0 +1,4 @@
+
+<p> <?= __('To finish payment print code below. You can make payment in any Tesco Lotus Store. Code is valid in next <b>24 hours only</b>. After that time your order will be cancelled.'); ?></p>
```

Comment:
English needs fixing
```
To finish payment, print the code below and bring it to make payment in any Tesco Lotus Store. Code is valid for the next <b>24 hours only</b>. After that time your order will be cancelled.'); ?></p>
```
`Cancelled` probably also needs to be changed to `Canceled` if we are using US English